src/data/faq.json

```diff
@@ -623,7 +623,19 @@
             {
                 "title": "Risk Assessment",
                 "id": "risk_assessment",
-                "accordion": [
+                "accordion": [
+                    {
+                        "title": "Dissimilar indication of risk status",
+                        "anchor": "dissimilar_indication_of_risk_status",
+                        "active": true,
+                        "textblock": [
+                            "Why do differences in infection risk status appear in different CWA app's, even for close-by smartphones?",
```

Comment:
app's => apps
(plural does not use an apostrophe)

Reply:
ok !